lib/adhearsion-i18n.rb

```diff
@@ -3,6 +3,7 @@
 %w{
   version
   plugin
+  formatter
```

Comment:
This should really just be required where it's used, in `call_controller_methods.rb`.